site/server/app/com/fortysevendeg/exercises/app.scala

```diff
@@ -7,6 +7,7 @@ import com.fortysevendeg.exercises.services.free._
 
 object app {
   type ExercisesAndUserOps[A] = Coproduct[ExerciseOp, UserOp, A]
+  type UserAndUserProgressOps[A] = Coproduct[UserOp, UserProgressOp, A]
```

Comment:
These are not reusable and won't be composed outside this context AFAIK so they may as well be called ModuleName01...